### ISSUE TITLE
Support for replacing MERGEFIELD with a table

### ISSUE THREAD

**Issue by ventoroai** [chrisfcarroll/MailMerge#12]
Is it possible to replace a MERGEFIELD with a table? If not supported in the present version, how do you think I can achieve this?

**Comment by chrisfcarroll** [comment]
No, not possible because the values for replacement are in a Dictionary<string, string>, so you can only replace mergefields with plain text.

The starting point for you to do it would be, I think, to spend enough hours (or days!) on e.g. [MS Learn on OOXML for wordprocessing](https://learn.microsoft.com/en-us/office/open-xml/word/overview) and/or [The Spec](https://ecma-international.org/publications-and-standards/standards/ecma-376/) and/or the much more readable [officeopenxml.com](http://officeopenxml.com) — in fact that website is probably the least daunting place to start — and/or stepping though some examples in [Learn.Some.OpenXml](https://github.com/chrisfcarroll/MailMerge/tree/main/Learn.Some.OpenXml.Sdk) in your debugger to get a feel for to work with a word XML document.

One task would be to work out whether or not the mergefield part of the spec lets you insert tables — I suspect not! But that doesn't really matter —  you could use [search-and-replace](https://github.com/chrisfcarroll/MailMerge/blob/main/Learn.Some.OpenXml.Sdk/Word2010SdtExamples.cs) functionality instead.
